reimbursement.md

```diff
@@ -498,24 +504,24 @@ Reimbursement requests can be retried via API or dashboard.
 These errors might be returned in the [request](https://fidel.uk/docs/reimbursement/#request) endpoint.
 
 <ReimbursementTable
-  headings={['HTTP Status Code', 'Error Code', 'Error Message']}
-  rows={[
-    ['400', 'reimbursement-account-network-inactive', 'Account cannot issue reimbursement for card network'],
-    ['400', 'reimbursement-account-not-enough-funds', 'Account does not have enough funds for issuing reimbursement'],
-    ['404', 'reimbursement-account-not-found', 'Account does not exist'],
-    ['400', 'reimbursement-already-created', 'Transaction reimbursement already created'],
-    ['400', 'reimbursement-amount-above-limit', 'Reimbursement amount is above the allowed limit'],
-    ['400', 'reimbursement-amount-greater-original-amount', 'Reimbursement amount is greater than original transaction amount'],
-    ['400', 'reimbursement-invalid-transaction', 'Transaction is invalid to issue reimbursement'],
-    ['500', 'reimbursement-network-internal-error', 'Network responded with internal error'],
-    ['404', 'reimbursement-transaction-not-found', 'Transaction does not exist'],
-    ['400', 'reimbursement-unsupported-network', 'Transaction network not supported for reimbursement'],
-    ['400', 'reimbursement-unsupported-currency', 'Transaction currency not supported for reimbursement'],
-    ['401', 'reimbursement-not-activated', 'The reimbursement product is not activated for this account'],
-    ['400', 'reimbursement-time-limit-after-transaction', 'Surpassed the time limit after the original transaction to issue a reimbursement'],
-    ['400', 'reimbursement-visa-invalid-community-code', 'The transaction community code is invalid'],
-    ['400', 'credits-account-not-enough-funds', 'Account does not have enough credits'],
-  ]}
+headings={['HTTP Status Code', 'Error Code', 'Error Message']}
+rows={[
```

Comment:
the indentation in this code seems to be helpful. Why was it removed?

Reply:
I'm not sure, maybe it was my Prettier config